db/migrate/20210813114619_add_name_to_device.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddNameToDevice < ActiveRecord::Migration[6.0]
+  def change
+    add_column :devices, :name, :string
```

Comment:
I added name for Devices because it's better for the association with the Sensor to pick the Device you want using 'name' over 'description'